src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebResponseObject.Common.cs

```diff
@@ -64,79 +51,46 @@ public long RawContentLength
         /// </value>
         public string RawContent { get; protected set; }
 
-        #endregion Properties
-
-        #region Methods
-
         /// <summary>
-        /// Reads the response content from the web response.
+        /// Gets the length (in bytes) of <see cref="RawContentStream"/>.
         /// </summary>
-        private void InitializeContent()
-        {
-            this.Content = this.RawContentStream.ToArray();
-        }
-
-        private static bool IsPrintable(char c)
+        public long RawContentLength
         {
-            return (char.IsLetterOrDigit(c) || char.IsPunctuation(c) || char.IsSeparator(c) || char.IsSymbol(c) || char.IsWhiteSpace(c));
+            get => RawContentStream is null ? -1 : RawContentStream.Length;
         }
 
         /// <summary>
-        /// Returns the string representation of this web response.
+        /// Gets the response body content as a <see cref="MemoryStream"/>.
         /// </summary>
-        /// <returns>The string representation of this web response.</returns>
-        public sealed override string ToString()
+        public MemoryStream RawContentStream
```

Comment:
`RawContentStream` (and maybe others) can be an auto-implemented property (removing the field).

Reply:
Done for `RawContentStream`, I will do the same for `Headers`